src/SFA.DAS.ApprenticeCommitments.Api/appsettings.json

```diff
@@ -12,4 +12,4 @@
   "Environment": "DEV",
   "Version": "1.0",
   "APPINSIGHTS_INSTRUMENTATIONKEY": ""
-}
```

Comment:
Can we lose this change? we shouldn't really be making changes to other teams projects within our own PR's